clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderProducedRecordContext.java

```diff
@@ -56,9 +60,9 @@ public class LeaderProducedRecordContext implements Measurable {
   private final Object valueUnion;
 
   /**
-   * This is the offset at which the message was produced in the Version Topic.
+   * This is the position at which the message was produced in the Version Topic.
    */
-  private long producedOffset = -1;
+  private PubSubPosition producedPosition = PubSubSymbolicPosition.LATEST;
```

Comment:
Should it be Position.EARLIEST instead?

Reply:
Good point. This was just a placeholder position, but I’ve updated it to use NO_UPSTREAM_POSITION instead, per our offline discussion, to avoid confusion.